src/strawchemy/dto/base.py

```
@@ -463,17 +463,18 @@ def _resolve_relation_type(
 
         if (is_type_hint_optional(type_hint) and not field.complete) or field.partial:
-            return Optional[dto]
+            return Optional[dto]  # ty: ignore[invalid-type-form]
         return dto
 
     def _resolve_type(
         self,
         field: DTOFieldDefinition[ModelT, ModelFieldT],
         dto_config: DTOConfig,
         node: Node[Relation[ModelT, DTOBaseT], None],
+        *_: Any,
```

Ruff (0.15.14)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*_`

(ANN401)